miner/src/block_assembler.rs

```diff
@@ -452,7 +452,7 @@ impl BlockAssembler {
         let occupied_capacity = output.occupied_capacity(Capacity::bytes(data.len())?)?;
 
         if reward < occupied_capacity {
-            return Err(TransactionError::InsufficientCellCapacity.into());
+            Err(TransactionError::OccupiedOverflowCapacity)?;
```

Comment:
the change from `InsufficientCellCapacity` to `OccupiedOverflowCapacity` is intended?

Reply:
I have renamed `InsufficientCellCapacity` -> `OccupiedOverflowCapacity `
https://github.com/nervosnetwork/ckb/pull/1344/files#diff-9f6a2acd3643aa4c24fdbf8922dbe532R6

Please tell me if you have better names or error descriptions!